spec/paypal_express/remote/build_plugin_helpers.rb

```diff
@@ -0,0 +1,38 @@
+module BuildPluginHelpers
+  def build_start_paypal_plugin(account_id = nil)
+    if account_id.nil?
+      plugin = build_plugin(::Killbill::PaypalExpress::PaymentPlugin, 'paypal_express')
+      start_plugin plugin
+    else
+      account_line = "  - :account_id: #{account_id}"
+      Dir.mktmpdir do |dir|
```

Comment:
You could greatly simply this. Take a look at the [YAML](http://ruby-doc.org/stdlib-2.1.0/libdoc/yaml/rdoc/YAML.html) API.